tests/transports-governance/conftest.py

```
@@ -0,0 +1,476 @@
+
+    def request(self, method: str, endpoint: str, **kwargs) -> requests.Response:
+        """Make HTTP request with comprehensive error handling"""
+        url = f"{self.base_url}/{endpoint.lstrip('/')}"
+        try:
+            response = self.session.request(method, url, **kwargs)
+            return response
+        except requests.exceptions.RequestException as e:
+            pytest.fail(f"Request failed: {method} {url} - {str(e)}")
+
+    # Budget operations
+    def create_budget(self, max_limit: int, reset_duration: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)